libcxx/docs/ReleaseNotes/20.rst

```diff
@@ -69,6 +69,10 @@ Improvements and New Features
 - The ``_LIBCPP_ABI_BOUNDED_ITERATORS_IN_STD_ARRAY`` ABI configuration was added, which allows storing valid bounds
   in ``std::array::iterator`` and detecting OOB accesses when the appropriate hardening mode is enabled.
 
+- The `assign(_InputIterator, _InputIterator)` function of `std::vector<_Tp, _Allocator>` has been optimized for
+  non-trivial element types, such as `std::vector<std::string>`, with a performance improvement of up to 2.3x. The
+  performance for trivial types, such as `std::vector<int>`, remains similar or shows slight improvements.
```

Comment:
Can we call out that this is _only_ for input iterators? This makes it look like it's for any kind of iterator.

Reply:
Thank you for your feedback! You're right. I should clarify that this optimization is for input iterators only. I will update the release note to reflect that.

Reply:
I will update the release note after all other comments have been addressed. By that time, I will be in a better position to make the update.